Makefile

```diff
@@ -611,12 +613,13 @@ podman-release.tar.gz: binaries docs  ## Build all binaries, docs., and installa
 	tar -czvf $@ --xattrs -C "$(TMPDIR)" "./$(SUBDIR)"
 	-rm -rf "$(TMPDIR)"
 
-podman-remote-release-%.zip: podman-remote-% install-podman-remote-%-docs  ## Build podman-remote for GOOS=%, docs., and installation zip.
+podman-remote-release-%_$(GOARCH).zip: podman-remote-% install-podman-remote-%-docs  ## Build podman-remote for GOOS=%, docs., and installation zip.
```

Comment:
Do we have a target to cross-build AARCH64 for OS X? I think we need one if not, we want to start distributing it.

Reply:
That's how I read Dan's question.  I already opened a jira card to explore adding it, and assigned myself.

Reply:
That is how I meant it.